swagger-spec/preprints/detail.yaml

```
@@ -32,10 +32,15 @@ get:
         application/json:
           data:
             relationships:
-              node:
+              contributors:
                 links:
                   related:
-                    href: https://api.osf.io/v2/nodes/bnzx5/
+                    href: https://api.osf.io/v2/contributors/bnzx5/
```

Should this be the following?
```suggestion
                    href: https://api.osf.io/v2/preprints/bnzx5/contributors/
```